paywall/src/data-iframe/index.2.0.ts

```diff
@@ -0,0 +1,45 @@
+import Postmate from 'postmate'
+import { Web3Service } from '@unlock-protocol/unlock-js'
+import { BlockchainHandler } from './BlockchainHandler'
+import { BlockchainDataStorable } from './BlockchainDataStorable'
+import configure from '../config'
+
+const {
+  readOnlyProvider,
+  unlockAddress,
+  blockTime,
+  requiredConfirmations,
+} = configure()
+const web3Service = new Web3Service({
+  readOnlyProvider,
+  unlockAddress,
+  blockTime,
+  requiredConfirmations,
+})
+
+// Start with a null object, it will be replaced when the authenticate
+// method in the model is called.
+let blockchainHandler = new BlockchainDataStorable()
+
+const authenticate = (lockAddresses: string[], accountAddress: string) => {
```

Comment:
the naming of this method feels a bit weird or appears a bit weird given the parameters being passed. Would we need to pass lock addresses here?

Reply:
Maybe `init` is a better choice -- those are the 2 pieces of data required to instantiate the handler:
- the list of lock addresses from the paywall config
- the account address for the user

Reply:
In fact, since there is a separation between the read and write operations, it's probably even better to disambiguate exactly what we're starting here.